Automattic/jetpack
# Forms: Add response inbox filters

## Proposed changes:

* Adds the Date and Source dropdown filters to the Responses inbox view

<img width="627" alt="image" src="https://user-images.githubusercontent.com/7811225/227572018-b83da668-3757-4e27-98a6-659b528f737f.png">


### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the jetpack forms package and the new dashboard before testing with:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to `/wp-admin/admin.php?page=jetpack-forms`
- Check if the "Date" and "Source" filters work properly
